packages/quantic/force-app/examples/main/lwc/exampleQuanticCategoryFacet/exampleQuanticCategoryFacet.js

```diff
@@ -0,0 +1,77 @@
+import {api, LightningElement, track} from 'lwc';
+
+export default class ExampleQuanticFacet extends LightningElement {
+    @api engineId = 'quantic-category-facet-engine';
+    @track config = {};
+    isConfigured = false;
+
+    pageTitle = 'Quantic Category Facet';
+    pageDescription = 'The Quantic Category Facetis a list of values for a certain field occurring in the results, ordered using a configurable criterion.'
+    options = [
+        {
+            attribute: 'field',
+            label: 'Field',
+            description: 'The name of the field to display as a facet.',
+            defaultValue: 'geographicalhierarchy'
+        },
+        {
+            attribute: 'label',
+            label: 'Label',
+            description: 'The label to use as the facet title.',
+            defaultValue: 'Country'
+        },
+        {
+            attribute: 'basePath',
+            label: 'BasePath',
+            description: 'The base path shared by all facet values, separated by commas.',
+            defaultValue: ''
+        },
+        {
+            attribute: 'noFilterByBasePath',
+            label: 'No Filter By Base Path',
+            description: 'Whether not to use the `basePath` as a filter for the results.',
+            defaultValue: false
+        },
+        {
+            attribute: 'noFilterFacetCount',
+            label: 'No Filter Facet Count',
+            description: 'Whether not to exclude the parents of folded results when estimating the result count for each facet value.',
```

Comment:
Maybe it is just me, but I have a hard time understanding what should happen when setting this option :sweat_smile:

The `Whether not to exclude...` part is difficult to understand, and the relation between `filter` and `parents of folded results` is unclear.

Using `Whether to include the parents...` would help make the description clearer, even though the description still seem unrelated to the attribute name.